lib/index.scss

```diff
@@ -25,6 +25,11 @@
 @import "./styles/editor/emoji";
 @import "./styles/editor/media-editor";
 
+//File Attachment
+@import "./styles/components/file-attachment";
+
+
+
```

Comment:
Configure your CSS/SCSS linter for VSCode. These blank lines should not come up.